docs/usage/agent-identity.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+parent: Usage
+---
+
+# Agent identity
+
+Zincati agent tries to derive an unique identity for the machine it is running on, by introspecting the underlying OS and reading user configuration.
+This includes assigning an ID and a group label specific to the agent, so that cluster-wide upgrades can be orchestrated via [phased rollouts][phased] and [lock-based][fleetlock-strategy] reboots.
+
+[phased]: auto-updates.md#phased-rollouts-client-wariness-canaries
+[fleetlock-strategy]:  updates-strategy.md#lock-based-strategy
+
+## Identity configuration
+
+All agent identity values are normally auto-detected at startup, and do not require user intervention.
+
+However, the following settings can be overridden through configuration fragments in the `identity` section:
+ * `group`: group label, used for graph fetching ([Cincinnati][cincinnati]) and reboot orchestration ([FleetLock][fleetlock])
+ * `node_uuid`: agent ID, used for graph fetching ([Cincinnati][cincinnati]) and reboot orchestration ([FleetLock][fleetlock])
+ * `rollout_wariness`: agent wariness to [phased rollouts][phased], used for graph fetching ([Cincinnati][cincinnati]).
+
+By default, the agent ID is automatically derived from `/etc/machine-id` in an application-specific way, group label is set to `default`, and rollout wariness is unset (Cincinnati backend will assign a dynamic value to each request).
```

Comment:
Something like this may work a bit better.

```suggestion
The following are defaults for each setting:
- `group` (group label) is set to `default`
- `node_uuid` (agent ID) is automatically derived from `/etc/machine-id` in an application-specific way
- `rollout_wariness` is unset and the Cincinnati backend will assign a dynamic value to each request
```